components/Instantsearch.jsx

```diff
@@ -0,0 +1,5 @@
+import { createInstantSearch } from 'react-instantsearch/server';
```

Comment:
remove if we get rid of instantsearch

Reply:
The UCSC pages still uses instant search. Unless you want to get rid of UCSC page as well